tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/error/ErrorTest.java

```diff
@@ -84,8 +84,8 @@ public void testIndirectErrorCtor() {
         Object returns = BRunUtil.invoke(errorTestResult, "testIndirectErrorConstructor");
         BArray errors = (BArray) returns;
         Assert.assertEquals(errors.size(), 4);
-        Assert.assertEquals(errors.get(0).toString(), "error UserDefErrorTwoA (\"arg\",message=\"\",data={})");
-        Assert.assertEquals(errors.get(1).toString(), "error UserDefErrorTwoA (\"arg\",message=\"\",data={})");
+        Assert.assertEquals(errors.get(0).toString(), "error UserDefErrorTwoA (\"arg\",data={},message=\"\")");
```

Comment:
Can you please mention (as a reply here) what kind of order changes we get with this change?

Reply:
Ping on this. Can we check if we can avoid the order change?